lib/rules/jsx-prefer-fragment-wrappers.js

```diff
@@ -35,23 +33,15 @@ module.exports = {
   },
 };
 
-function isPascalCase(string) {
-  return string === pascalCase(string);
+function isAcceptedTagName({openingElement: {name}}) {
+  return ['div', 'span'].every((tagName) => tagName !== name.name);
```

Comment:
IMO `![].some` is probably more readable here, or inverting the whole function's purpose, it is hard for me to read "accepted tag name, but we are checking that ever non-accepted tag name is not the case"

Reply:
I like `![].some` and keeping this purpose consistent with the others like it in the file.